### DISCUSSION TITLE
Which OS should it used?

### DISCUSSION THREAD

**Discussion by zongbb1983** [pallets/flask#5417] [Q&A]
Which OS should it used? Windows or Linux? or something else? For I used it under the Windows system, this error occured.

Traceback (most recent call last):
  File "C:\Users\jack\PycharmProjects\flask-main\src\flask\app.py", line 7, in <module>
    import typing as t
  File "C:\Users\jack\PycharmProjects\flask-main\src\flask\typing.py", line 5, in <module>
    if t.TYPE_CHECKING:  # pragma: no cover
AttributeError: partially initialized module 'typing' has no attribute 'TYPE_CHECKING' (most likely due to a circular import)

**Comment by CheeseCake87** [comment]
You need to change the name of your `typing.py` file to something other than the name of the package you are trying to use.

**Comment by mikebionic** [comment]
Try adding dot before package

`import .typing as t`
